hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamerTestBase.java

```diff
@@ -634,7 +634,7 @@ static void waitTillCondition(Function<Boolean, Boolean> condition, Future dsFut
         boolean ret = false;
         while (!ret && !dsFuture.isDone()) {
           try {
-            Thread.sleep(3000);
+            Thread.sleep(2000);
```

Comment:
any reason for this change?

Reply:
it could reduce the total test run time. So reduced the wait time.

Reply:
I mean, nothing specific to tests we added. just happened to notice and fixed it